content/client-server-api/modules/voip_events.md

```diff
@@ -171,18 +171,34 @@ In response to an incoming invite, a client may do one of several things:
 
 ##### Streams
```

Comment:
I am not a fan of the formulation in places. For example there are several "i.e." which should not be needed if the sentences were reworked.

However there were no comments about it in the first review, so maybe it was not deemed necessary. Should I still take a crack at it?

Reply:
I think the wording is *okay* as it stands, but there is certainly room for improvement. If you'd like to do so, that would be great!

Reply:
I have reworked part of the text.